forge-gui/res/cardsfolder/upcoming/tasha_unholy_archmage.txt

```diff
@@ -0,0 +1,12 @@
+Name:Tasha, Unholy Archmage
+ManaCost:2 U B
+Types:Legendary Planeswalker Tasha
+Loyalty:4
+A:AB$ Effect | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | Triggers$ AttackTrigger | Duration$ UntilYourNextTurn | Name$ Tasha, Unholy Archmage's +1 Effect | AILogic$ Main2 | SpellDescription$ Until your next turn, whenever a creature attacks you or CARDNAME, put a -1/-1 counter on that creature.
```

Comment:
This Name$ param is unneeded – since this card only makes one effect, no need to be specific, the defaults will be fine